integration-tests/baas-test-server/cli.ts

```diff
@@ -95,12 +99,19 @@ yargs(hideBin(process.argv))
       docker.ensureNoBaas();
 
       if (argv.githash) {
-        docker.spawnBaaS({ tag: argv.githash, accessKeyId: AWS_ACCESS_KEY_ID, secretAccessKey: AWS_SECRET_ACCESS_KEY });
+        docker.spawnBaaS({
+          image: argv.githash,
+          accessKeyId: AWS_ACCESS_KEY_ID,
+          secretAccessKey: AWS_SECRET_ACCESS_KEY,
+        });
+      } else if (argv["latest-local"]) {
+        const id = docker.getLatestLocalId();
+        docker.spawnBaaS({ image: id, accessKeyId: AWS_ACCESS_KEY_ID, secretAccessKey: AWS_SECRET_ACCESS_KEY });
       } else {
         const tag = await docker.fetchBaasTag(argv.branch);
         assert(AWS_PROFILE, "Missing AWS_PROFILE env");
```

Comment:
Would it make sense to put this `assert` before fetching?

Reply:
Perhaps 🤔 I think I like this to be next to the line that relies on it, to increase the likelihood that they're moved together in a future refactoring.

Reply:
🙂